### PR TITLE
Create Rule “do-you-use-emoji-in-dynamics-label/rule”

### PR DESCRIPTION
**Pull request by zl24622**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/do-you-use-emoji-in-dynamics-label/rule.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+### Option 1 -  Description attribute
+
+Advantages:
+
+* Out of box feature.  No customizations needed.
+
+Disadvantages:
+
+* Not obvious to see - user normally doesn't aware to hover to the label to see the note.
```
Comment (calumjs) [inline rules/do-you-use-emoji-in-dynamics-label/rule.md:26]:
```suggestion
* Not obvious to see - users are often unaware they can hover on the label to see the note.
```

File: rules/do-you-use-emoji-in-dynamics-label/rule.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Disadvantages:
+
+* Need more time to develop or install a package.
+* Introduce a dependent package to the system.
+
+![Figure: Use a PCF control to show a note and link to a web page](screenshot-at-oct-28-11-23-52.png "Figure: Use a PCF control to show a note and link to a web page")
+
+### Option 3 -  Description field + Emoji (Recommended)
+
+Add a emoji in the label to remind people to hover to see the tooltip.
```
Comment (calumjs) [inline rules/do-you-use-emoji-in-dynamics-label/rule.md:47]:
```suggestion
Add an emoji in the label to remind people to hover to see the tooltip. 
```

File: rules/do-you-use-emoji-in-dynamics-label/rule.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+* Need more time to develop or install a package.
+* Introduce a dependent package to the system.
+
+![Figure: Use a PCF control to show a note and link to a web page](screenshot-at-oct-28-11-23-52.png "Figure: Use a PCF control to show a note and link to a web page")
+
+### Option 3 -  Description field + Emoji (Recommended)
+
+Add a emoji in the label to remind people to hover to see the tooltip. 
+
+![Figure: Add a emoji on the label to remind to people it's a tooltip ](screenshot-at-oct-28-11-29-40.png "Figure: Add a emoji on the label to remind to people it's a tooltip ")
```
Comment (calumjs) [inline rules/do-you-use-emoji-in-dynamics-label/rule.md:49]:
```suggestion
![Figure: Add an emoji on the label to remind to people it's a tooltip ](screenshot-at-oct-28-11-29-40.png "Figure: Add a emoji on the label to remind to people it's a tooltip ")
```

File: rules/do-you-use-emoji-in-dynamics-label/rule.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+![Figure: The out-of-box description attribute can be used as a tooltip. But it's not obvious to see.](screenshot-at-oct-28-11-17-04.png "Figure: The out-of-box description attribute can be used as a tooltip. But it's not obvious to see.")
+
+
+
+### Option 2 -  PCF control
+
+Advantages:
+
+* Flexibility - allow to customize any style or behaviour.
```
Comment (calumjs) [inline rules/do-you-use-emoji-in-dynamics-label/rule.md:36]:
```suggestion
* Flexibility - can customize any style or behaviour.
```

**Review by calumjs — CHANGES_REQUESTED** [review]
Just fix a couple of minor typos then good to go

**Review by zl24622 — COMMENTED** [review]
approved

**Review by UlyssesMaclaren — APPROVED** [review]

**Review by adamcogan — APPROVED** [review]

**Comment by bradystroud** [comment]
@calumjs this one is blocked by you
Please re-review

**Review by calumjs — APPROVED** [review]

**Comment by calumjs** [comment]
🚢

**Pull request merged**